docs/topics/i18n/translation.txt

```diff
@@ -1546,6 +1540,8 @@ Here's example HTML template code:
     {% csrf_token %}
```

Comment:
Interesting. I assumed it was there because the `{% get_language_info_list %} tag was already in the template.